net/openfortivpn/files/openfortivpn.init

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh /etc/rc.common
+
+START=60
```

Comment:
Use 99 instead unless you have a very good reason for it to start this early in the boot process